test/exnumerable_test.exs

```diff
@@ -66,4 +72,41 @@ defmodule ExnumeratorTest do
     assert MessageAsAtom.load(:invalid)    == :error
     assert MessageAsAtom.dump(:invalid)    == :error
   end
+
+  describe "keywords" do
+    test "should store given values" do
+      assert MessageAsKeywords.values() == @values_keywords
+    end
+
+    test "should return a random value" do
+      assert MessageAsKeywords.sample() in @values_keywords
+    end
+
+    test "should cast atom, string, or raw value" do
+      assert MessageAsKeywords.cast(:sent) == {:ok, :sent}
+      assert MessageAsKeywords.cast("sent") == {:ok, :sent}
+      assert MessageAsKeywords.cast("S") == {:ok, :sent}
+    end
+
+    test "should not cast non-whitelisted values" do
+      assert MessageAsKeywords.cast(:invalid) == :error
+      assert MessageAsKeywords.cast("invalid") == :error
+    end
+
+    test "should dump atom, string, or raw value" do
+      assert MessageAsKeywords.dump(:sent) == {:ok, "S"}
+      assert MessageAsKeywords.dump("sent") == {:ok, "S"}
+      assert MessageAsKeywords.dump("S") == {:ok, "S"}
+    end
+    test "should not dump non-whitelisted values" do
+      assert MessageAsKeywords.dump(:invalid) == :error
+      assert MessageAsKeywords.dump("invalid") == :error
+    end
+
+    test "should load only the raw value" do
+      assert MessageAsKeywords.load("S") == {:ok, :sent}
+      assert MessageAsKeywords.load(:sent) == :error
+      assert MessageAsKeywords.load("sent") == :error
+    end
+  end
```

Comment:
Please align `==`.

Reply:
@KamilLelonek anything related to formatting, we should Elixir to do it job by `mix format` IMHO